playbooks/common/openshift-checks/health.yml

```diff
@@ -1,14 +1,9 @@
 ---
-- include: ../openshift-cluster/std_include.yml
-  tags:
-  - always
-
 - name: Run OpenShift health checks
-  hosts: OSEv3
+  hosts: oo_all_hosts
   roles:
-  - openshift_health_checker
-  vars:
-  - r_openshift_health_checker_playbook_context: health
+  - role: openshift_health_checker
+    r_openshift_health_checker_playbook_context: health
```

Comment:
I'm not sure I understand your first sentence.  The play has a `vars:` section and a `roles:` section.  The `vars:` apply to the entire play.  If this play were to have more roles or other sections added in the future, such as 'pre-tasks:`, `tasks:`, then it could be unclear as to where those vars actually need to be specified.  Putting the vars with the call for the role makes it clear that they need to be set for the role.